plugins/linux-capture/xshm-input.c

```diff
@@ -346,10 +346,9 @@ static bool xshm_server_changed(obs_properties_t *props, obs_property_t *p,
 	bool randr = randr_is_active(xcb);
 	bool xinerama = xinerama_is_active(xcb);
 	int_fast32_t count =
-		(randr) ? randr_screen_count(xcb)
-			: (xinerama)
-				  ? xinerama_screen_count(xcb)
-				  : xcb_setup_roots_length(xcb_get_setup(xcb));
+		randr ? randr_screen_count(xcb)
+		      : (xinerama ? xinerama_screen_count(xcb)
+				  : xcb_setup_roots_length(xcb_get_setup(xcb)));
```

Comment:
This change seems unrelated.  Was there some sort of compiler warning?  Keep in mind that CI uses clang-format 10 at the moment.

Reply:
VS clang-format altered it, so I messed with it a little. I'll fix it from the Linux side in a bit.

Reply:
Actually, Linux is fine, and CI was complaining about the other file. This should be good to go.

Reply:
Looks like VS2019 currently ships with clang-format 11.0.0.0, which we don't use on CI because it produces different output from clang-format 10 with the same settings.